### ISSUE TITLE
same term both preferred and alternate

### ISSUE THREAD

**Issue by maglott** [obophenotype/human-phenotype-ontology#7174]
**HPO term/description**
We are finding many HPO ids that appear to have the same term for preferred and alternate labels.  
e.g. https://hpo.jax.org/app/browse/term/HP:0000105

Is this expected?

**Comment by matentzn** [comment]
I think @pnrobinson HPO is doing the right thing here, even though it may be slightly confusing. The preferred label (we use rdfs:label to encode this) is the single unique string by which we refer to the phenotype. The synonyms (alternative terms) have many different use cases, like "layperson", "akronym" and "preferred by NIH"  and as such can overlap with the label. They are fulfill very different use cases. 

I would recommend to users to either:
1) show the contextual information (layperson, acronym, broad synonym) to their users or
2) filter then out on the client side (hiding them from the end user if they equal the label)


There may be one exception to this:

If we have an exact synonym that is unqualified (I.e no source, no layperson, etc) than I think we should consider deleting it from HPO, as this seems purely redundant.

**Comment by matentzn** [comment]
I think its the right way to do it..

**Comment by pnrobinson** [comment]
@maglott  -- please see above. This is the only way for us to encode that the primary label is also a layperson term. Our Q/C would pick up duplications otherwise.
